src/redux/thunks/loadPolicies.ts

```diff
@@ -0,0 +1,220 @@
+import {createAsyncThunk} from '@reduxjs/toolkit';
+
+import {v4 as uuid} from 'uuid';
+
+import {AlertEnum} from '@models/alert';
+import {BasicPolicy, POLICY_VALIDATOR_MAP, Policy, PolicyConfig, ValidatorId} from '@models/policy';
+
+import {setAlert} from '@redux/reducers/alert';
+import {loadBinaryResource} from '@redux/services';
+
+import featureJson from '@src/feature-flags.json';
+
+import {loadPolicy} from '@open-policy-agent/opa-wasm';
+
+export const loadPolicies = createAsyncThunk<Policy[]>('main/loadPolicies', async (_, {dispatch}) => {
+  if (!featureJson.ResourceScanning) {
+    return [];
+  }
+
+  try {
+    const plugin = DEFAULT_TRIVY_PLUGIN;
+    const wasm = loadBinaryResource('policies/bundle/policy.wasm');
+
+    if (!wasm) {
+      throw new Error('policy not found');
+    }
+
+    const validatorId: ValidatorId = uuid();
+    const validator = await loadPolicy(wasm);
+    POLICY_VALIDATOR_MAP[validatorId] = validator;
+
+    return [{validatorId, metadata: plugin, config: DEFAULT_CONFIGURATION}];
+  } catch (err) {
+    dispatch(
+      setAlert({
+        title: 'Failed to load policy',
+        message: (err as Error).message,
+        type: AlertEnum.Warning,
+      })
+    );
+    return [];
+  }
+});
+
+const DEFAULT_CONFIGURATION: PolicyConfig = {enabled: true};
+
+const DEFAULT_TRIVY_PLUGIN: BasicPolicy = {
```

Comment:
Not sure yet whether we want rule metadata embedded in the Rego policies or within our plugin configuration. I decided that it's not relevant for now and we can refactor it later. This was the easiest so I went with it.